cEP-0003.md

```diff
@@ -119,10 +124,16 @@ Electing Team Leaders
 Any coala member may propose him or herself to be a Team Leader. The
 proposal should be public and include reasons and plans to take that
 particular team forward. (Archived in the wiki.)
+It is also possible for every coala member to propose another coala
+member as a Team Leader.
```

Comment:
I'd rather include that team leader and member applications are done via the teams repo publically?